src/scanpy/preprocessing/_pca/__init__.py

```diff
@@ -191,7 +194,9 @@ def pca(  # noqa: PLR0912, PLR0913, PLR0915
     `.obsm['X_pca' | key_added]` : :class:`~scipy.sparse.csr_matrix` | :class:`~scipy.sparse.csc_matrix` | :class:`~numpy.ndarray` (shape `(adata.n_obs, n_comps)`)
         PCA representation of data.
     `.varm['PCs' | key_added]` : :class:`~numpy.ndarray` (shape `(adata.n_vars, n_comps)`)
-        The principal components containing the loadings.
+        The principal components containing the loadings *when `obsm=None`*.
+    `.uns['pca' | key_added]['components']` : :class:`~numpy.ndarray` (shape `(adata.obsm][obsm].shape[1], n_comps)`)
+        The principal components containing the loadings *when `obsm="..."`*.
```

Comment:
I think this is an OK alternative location for `.varm['PCs' | key_added]` when that’s impossible? Pretty complex but what can one do …

One way to make it a tad more consistent would be to use `'PCs'` instead of `'components'`.